ts/test/test-profiler.ts

```diff
@@ -276,9 +276,7 @@ describe('Profiler', () => {
 
       requestStub = sinon.stub(common.ServiceObject.prototype, 'request')
                         .onCall(0)
-                        .returns(new Promise(resolve => {
-                          resolve([{}, {statusCode: 200}]);
-                        }));
+                        .callsArgWith(1, null, {}, {statusCode: 200});
```

Comment:
This is great, but if you wanted to keep the promise workflow you could have used `.resolves(...)`